src/main/telemetry/crsf.c

```diff
@@ -265,7 +265,7 @@ void crsfFrameGps(sbuf_t *dst)
 Payload:
 int16_t     Vertical speed ( cm/s )
 */
-void crsfFrameVarioSensor(sbuf_t *dst)
+MAYBE_UNUSED static void crsfFrameVarioSensor(sbuf_t *dst)
```

Comment:
Maybe should be within `#ifdef USE_GPS` ?
I am not sure but seems to only be used when GPS is defined.

Reply:
I did consider it (there are more similar cases). But it will lead to logic duplication (this one is IMO USE_VARIO, but other other places would need testing of multiple defines).
It is still `static`, so scope is limited to single file only. I just hope that MAYBE_UNUSED won't get overused

Reply:
We can always count the occurrences and report in PRs as a check

Reply:
I'd wait a bit to see if it is a problem.

Ideally some static checker would test that MAYBE_UNUSED is actually used in some combination. Otherwise change it to LOCAL_UNUSED_FUNCTION (and do the same for unused public functions).
Then is is easy to list LOCAL_UNUSED_FUNCTION as cleanup TODO and decide if they should be removed.,